package.json

```diff
@@ -267,8 +274,26 @@
     "@swc/core": {
       "built": true
     },
+    "@vscode/codicons": {
+      "unplugged": true
```

Comment:
do we actually need these `unplug` or was it only for debugging purpose ?

Reply:
Assets are loaded from the node_modules at runtime so we kind of need to do it

Reply:
Not sure to understand the reply :S.
The `unplug` does not generate a `node_module` but a folder with a hash in the path.
I do not see the need.

(same fo the other `unplug` packages)